homeassistant/components/sensor/mold_indicator.py

```diff
@@ -19,7 +19,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_CRITICAL_TEMP = 'Est. Crit. Temp'
+ATTR_CRITICAL_TEMP = 'EstimatedCriticalTemp'
```

Comment:
Attributes should be lowercase snakecase.

Reply:
yes indeed please. Change `EstimatedCriticalTemp` in `estimated_critical_temp`

is this updated now, and do we need to change anything in the HA setup?
just using platform: most_indicator as is. 
No need to put mold_indicator.py in custom_components ?

Cheers,
Marius